dev/RadioButtons/InteractionTests/RadioButtonsTests.cs

```diff
@@ -142,7 +142,7 @@ public void FocusComingFromAnotherRepeaterTest()
                 VerifySelectedFocusedIndex(2, true);
 
                 // Move focus to the first RadioButtons control by pressing TAB again
-                KeyboardHelper.PressKey(Key.Tab);
+                KeyboardHelper.PressKey(Key.Tab, ModifierKey.Shift);
```

Comment:
Is this intentional ?

Reply:
Curious as to why you are removing the bit to hide test app bar?

Reply:
IIRC, you couldn't really leave the app after that since the only way to get back was using some off screen elements that are triggered through UIA when executing tests.